services/prometheus/prometheus.go

```diff
@@ -302,7 +301,14 @@ func (svc *Service) marshalConfig() ([]byte, error) {
 			cfg.GlobalConfig.EvaluationInterval = config.Duration(s.LR)
 		}
 
-		cfg.RuleFiles = append(cfg.RuleFiles, "/srv/prometheus/rules/*.rules.yml")
+		cfg.RuleFiles = append(
+			cfg.RuleFiles,
+
+			// That covers all .yml files, including:
+			// pmm.rules.yml managed by pmm-managed;
+			// user-supplied files.
+			"/srv/prometheus/rules/*.yml",
```

Comment:
That Go string literal is used in a single place. There no need to define a constant when it is not shared. Adding a name to it will not make it more clear.